Calculation.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalculatorLibrary
+{
+    public class Calculation
+    {
+        protected double num1 = 0;
+        protected double num2 = 0;
+        public double Result { get; protected set; } = 0;
+        protected string op = "";
```

Comment:
In general, refrain from abreviations when you name things. Always write a full word. Again readability. 🔍